filc/retrieval.go

```diff
@@ -342,13 +342,8 @@ func (node *Node) RetrieveFromBestCandidate(
 			continue
 		}
 
-		const progressLogInterval = time.Second / 5
-		lastProgressLog := time.Now()
 		stats_, err := fc.RetrieveContentWithProgressCallback(ctx, query.Candidate.Miner, proposal, func(bytesReceived uint64) {
-			if time.Since(lastProgressLog) > progressLogInterval {
-				lastProgressLog = time.Now()
-				fmt.Printf("%v (%v)\r", bytesReceived, humanize.IBytes(bytesReceived))
-			}
+			fmt.Printf("%v (%v)\r", bytesReceived, humanize.IBytes(bytesReceived))
```

Comment:
`xxx\r` stuff should generally go onto STDERR. You want `fmt.Fprintf( os.Stderr, ... )`